app/views/parks/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="container">
+  
+<h1><%= @park.name %></h1>
+<p><%= @park.description %></p>
+<img src='<%= @park.url %>' alt="">
```

Comment:
I like that you have the url feeding directly in to the image link but if the link isn't to an image or the URL is invalid it will break.